package.json

```diff
@@ -71,7 +72,7 @@
     "commander": "^2.8.1",
     "css-loader": "^0.15.1",
     "d3": "^3.5.6",
-    "elasticsearch": "^5.0.0",
+    "elasticsearch": "elastic/elasticsearch-js#master",
     "elasticsearch-browser": "^5.0.0",
     "expiry-js": "^0.1.7",
     "exports-loader": "^0.6.2",
```

Comment:
`angular-cookies` is in there...

As far as the others (`hapi-auth-cookie`, `btoa`, and `htpasswd`), I was under the impression we could have a package.json for each module (you'll notice that there is one in `src/server/auth/`, `src/plugins/esProxyAuth`, and `src/plugins/htpasswdAuth`). If you `cd` into these directories and do an `npm install`, it should work. But it seems like extra work to have to do this. So either I can move these dependencies out of their associated `package.json` files, or maybe we should consider some way of looking recursively at subdirectories for `package.json` files and installing dependencies from there too.

Or, maybe there's a way to do this already, and I'm just doing it wrong. Lemme know, @spalger and @rashidkpc.